src/librbd/image_watcher/Notifier.h

```diff
@@ -4,51 +4,17 @@
 #ifndef CEPH_LIBRBD_IMAGE_WATCHER_NOTIFIER_H
 #define CEPH_LIBRBD_IMAGE_WATCHER_NOTIFIER_H
 
-#include "include/int_types.h"
-#include "include/buffer_fwd.h"
-#include "include/Context.h"
-#include "common/Mutex.h"
-#include <list>
+#include "librbd/object_watcher/Notifier.h"
 
 namespace librbd {
 
 struct ImageCtx;
 
 namespace image_watcher {
 
-class Notifier {
+class Notifier : public object_watcher::Notifier {
```

Comment:
Minor: since this class doesn't appear to do anything anymore, can you just swap the current calls to image_watcher::Notifier to object_watcher::Notifier?